main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -1860,16 +1873,24 @@ object Weeder {
             traverse(elms) {
               case Pattern.Wild(loc) => WeededAst.RestrictableChoicePattern.Wild(loc).toSuccess
               case Pattern.Var(ident, loc) => WeededAst.RestrictableChoicePattern.Var(ident, loc).toSuccess
-              case other => WeederError.UnsupportedRestrictedChoicePattern(star, other.loc).toFailure
+              case other =>
+                val err = WeederError.UnsupportedRestrictedChoicePattern(star, other.loc)
+                WeededAst.RestrictableChoicePattern.Wild(other.loc).toSoftFailure(err)
             }
           case Pattern.Wild(loc) => List(WeededAst.RestrictableChoicePattern.Wild(loc)).toSuccess
           case Pattern.Var(ident, loc) => List(WeededAst.RestrictableChoicePattern.Var(ident, loc)).toSuccess
-          case other => WeederError.UnsupportedRestrictedChoicePattern(star, other.loc).toFailure
+          case other =>
+            val err = WeederError.UnsupportedRestrictedChoicePattern(star, other.loc)
+            List(WeededAst.RestrictableChoicePattern.Wild(other.loc)).toSoftFailure(err)
         }
         mapN(innerVal) {
           case inner => WeededAst.RestrictableChoicePattern.Tag(qname, inner, loc)
         }
-      case _ => WeederError.UnsupportedRestrictedChoicePattern(star, p0.loc).toFailure
+      case _ =>
+        val err = WeederError.UnsupportedRestrictedChoicePattern(star, p0.loc)
+        val pats = List(WeededAst.RestrictableChoicePattern.Wild(p0.loc))
+        val qname = Name.mkQName(new Name.SyntheticIdent(SourcePosition.Unknown, "Error" + Flix.Delimiter, SourcePosition.Unknown))
+        WeededAst.RestrictableChoicePattern.Tag(qname, pats, p0.loc).toSoftFailure(err)
```

Comment:
Lets undo this because the boys are doing all kinds of change to everything named `Restrictable*`.